src/encoded/schemas/functional_characterization_experiment.json

```diff
@@ -448,6 +448,12 @@
         "elements_references.crispr_screen_tiling": {
             "title": "CRISPR screen tiling"
         },
+        "elements_references.elements_selection_method": {
+            "title": "Elements selection method"
+        },
+        "elements_references.examined_loci": {
+            "title": "Reference loci"
+        },
```

Comment:
You’re bringing in an entire elements_references array objects here but looks like you only use `symbol` and maybe `@id`. To reduce the size of the search, I would just specify `elements_references.examined_loci.symbol` and a separate `elements_references.examined_loci.@id`.

Reply:
ok